### DISCUSSION TITLE
Last updates changed my image generation results (slightly)

### DISCUSSION THREAD

**Discussion by Abdel135** [lllyasviel/Fooocus#3373] [Q&A] [answered]
Hey guys, a couple hours ago i ran fooocus (colab version) and i noticed that my images are slightly different (which is important for me), i used the exact same sdxl checkpoint, prompt, seed, gfc, resolution .... like i always do. is it because of the last update or maybe the models that are automatically downloaded from huggingface have been updated very recently.
anyways, which branch with functional code that correspond to the main branch at least 10 days ago ?

**Comment by mashb1t** [comment, marked as answer]
This has to do with the model / LoRA updates. You can either choose to checkout 2.4.3 manually, then run launch.py instead of entry_with_update.py or create presets with the models / LoRAs of 2.4.3.

**Reply by Abdel135** [reply]
thanks a lot for the feedback and the constant upgrades, its turns out that the prompts are  line sensitive ('\n'), i didn't know that the prompt that i used previously had a newline in the middle, that was the only issue.
